jsettlers.common/src/main/java/jsettlers/common/position/IPositionSupplier.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
/* is the new style (which IntelliJ uses with its generator). I use the new one for new files but won't change it for old files (if IntelliJ isn't doing it because I move a file).